docs/how_to_contribute.rst

```diff
@@ -0,0 +1,62 @@
+
+Contributor Guide
+=================
+
+Step-by-step
+------------
+This guide will take you through the necessary steps in order to contribute code to the repository.
+
+
+1. Setup
+^^^^^^^^^^^^^^^^^^
+Fork the main repository, and clone your fork onto the machine of your choice.
```

Comment:
Need a reference for beginners.

Reply:
Let me move the link from further below up here?

Reply:
How is this?